packages/wonder-blocks-modal/components/two-column-modal.js

```diff
@@ -126,17 +132,16 @@ class TwoColumnModal extends React.Component<Props> {
                     mediaSize === "small" ? styles.smallDialog : styles.dialog
                 }
             >
-                <ContentWrapper {...this.props} />
+                <WrappedContentWrapper {...this.props} />
             </ModalDialog>
         );
     }
 }
 
-export default MediaLayoutWrapper(TwoColumnModal);
-
 const styles = StyleSheet.create({
     dialog: {
         width: "86.72%",
+        maxWidth: 960,
```

Comment:
This should be 888 to match the spec.

Reply:
The StandardModal dimensions are also off, they should be:
```
width: "93.75%",
maxWidth: 960,
height: "81.25%",
maxHeight: 624,
```

Reply:
🛑 this is blocking.

Reply:
Fixed, I got rid of the `minHeight: 200`